README.md

```diff
@@ -87,7 +87,7 @@ For Eclipse the process requires loading the formatting style and the import ord
 </details> 
 
 [openhospital]: https://www.open-hospital.org/
-[core]: https://github.com/informatici/openhospital/openhospital-core
+[core]: https://github.com/informatici/openhospital-core
```

Comment:
The problem was only here, in the reusable link (https://github.com/informatici/openhospital-core); with this fixed, you can leave unchannged the other ones.

Thanks!